src/services/ProfileApiService.js

```diff
@@ -131,30 +132,39 @@ export function patchPreferences(username, preferences) {
   });
 }
 
-export function getCourseCertificates(username) { // eslint-disable-line no-unused-vars
-  return new Promise((resolve, reject) => { // eslint-disable-line no-unused-vars
-    const dummyData = [
-      // {
-      //   type: {
-      //     key: 'micro_masters',
-      //     name: 'Micro Masters',
-      //   },
-      //   title: 'Microtonal Scales',
-      //   organization: 'New England Conservatory',
-      //   downloadUrl: 'https://pics.me.me/booplesnoot-36468371.png',
-      // },
-      // {
-      //   type: {
-      //     key: 'micro_masters',
-      //     name: 'Micro Masters',
-      //   },
-      //   title: 'Kazoo Pinch Harmonics',
-      //   organization: 'New England Conservatory',
-      //   downloadUrl: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR88selLYRHDQemO8MnY3w3ajlSuKZcoRhVTp3eGvKq2HMRTn8q',
-      // },
-    ];
-    setTimeout(() => {
-      resolve(dummyData);
-    }, 200);
+export function getCourseCertificates(username) {
+  const url = `${certificatesApiBaseUrl}/${username}/`;
+
+  return new Promise((resolve, reject) => {
+    // const dummyData = [
+    //   {
+    //     type: {
+    //       key: 'micro_masters',
+    //       name: 'Micro Masters',
+    //     },
+    //     title: 'Microtonal Scales',
+    //     organization: 'New England Conservatory',
+    //     downloadUrl: 'https://pics.me.me/booplesnoot-36468371.png',
+    //   },
+    //   {
+    //     type: {
+    //       key: 'micro_masters',
+    //       name: 'Micro Masters',
+    //     },
+    //     title: 'Kazoo Pinch Harmonics',
+    //     organization: 'New England Conservatory',
+    //     downloadUrl: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR88selLYRHDQemO8MnY3w3ajlSuKZcoRhVTp3eGvKq2HMRTn8q',
+    //   },
+    // ];
+    // setTimeout(() => {
+    //   resolve(dummyData);
+    // }, 200);
```

Comment:
Are we ready to delete the fake data?

Reply:
Given that the format of it isn't valid anymore, probably.